ci/tox/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN ln -s /usr/local/bin/python /usr/bin/python
 
 ARG git_version
 RUN set -x \
-    && apk add --no-cache --virtual git-build-deps  alpine-sdk autoconf gettext wget zlib-dev \
```

Comment:
Actually, `git-build-deps` is an arbitrary name of package group (for easier removal in `apk del git-build-deps`) and NOT an actual Alpine package